src/metabase/email.clj

```diff
@@ -95,12 +98,54 @@
                   (assert (#{:tls :ssl :none :starttls} (keyword new-value))))
                 (setting/set-value-of-type! :keyword :email-smtp-security new-value)))
 
+(defsetting email-rate-limit-per-second
+  (deferred-tru "The maximum number of emails that can be sent per second. The quotas are based on the total number of recipients")
```

Comment:
```suggestion
  (deferred-tru "The maximum number of emails, across all per recipients, that can be sent per second. <add something about the semantics where we let the email through anyway>")
```

Reply:
> The maximum number of emails, across all per recipients

=> The maximum number of recipients, across all emails

how about this?

Reply:
Even better!